lib/src/ruby/module.cc

```diff
@@ -113,7 +116,7 @@ extern "C" {
         set_level(log_level::warning);
 
         // Initialize ruby
-        auto ruby = facter::ruby::api::instance();
+        auto ruby = api::instance();
```

Comment:
We need to set `api::ruby_lib_location`, doing it before this call should be sufficient, based on FACTER_RUBY if defined.